lib/ohai/plugins/aix/virtualization.rb

```diff
@@ -91,7 +91,7 @@
                       }
               wpars[wpar_name][title][key] = value
             when "security settings"
-              privileges ||= ""
+              privileges = String.new
```

Comment:
to be clear, these aren't the same, you used to preserve any setting of priveleges, now you're deleting it.